x-pack/plugins/endpoint/public/applications/endpoint/store/managing/middleware.ts

```diff
@@ -27,13 +28,11 @@ export const managementMiddlewareFactory: MiddlewareFactory<ManagementListState>
     ) {
       const managementPageIndex = pageIndex(state);
       const managementPageSize = pageSize(state);
+      const body: MetadataIndexGetBodyInput = {
```

Comment:
This type is based on the schema. If the schema changes, this type will change as well.